Description: Downgrade textwrap dep to 0.11, same as clap 2.33.3
 Updating clap 2.33.3 to use textwrap 0.14 is much more painful
 .
 This patch can be dropped when debcargo updates clap to version 3
 which uses textwrap 0.14.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -74,7 +74,7 @@
 version = "3"
 
 [dependencies.textwrap]
-version = "0.14"
+version = "0.11"
 
 [dependencies.toml]
 version = "0.5"
